Assets/LeapMotionModules/ElementRenderer/Scripts/Features/Mesh/ProceduralMeshSource.cs

```diff
@@ -0,0 +1,16 @@
+﻿using UnityEngine;
+using UnityEngine.Rendering;
+
+[RequireComponent(typeof(LeapGuiElement))]
+public abstract class ProceduralMeshSource : MonoBehaviour {
+
+  // Just to make sure all procedural mesh sources can be enabled/disabled
+  protected virtual void Start() { }
+
+  /// <summary>
+  /// Tried so generate a procedural mesh for the given mesh feature. 
```

Comment:
Ah yeah that is a little unintuitive.  The system assumed that all uvs map to a texture, and so when the texture gets atlased, the uvs need to be remapped.  This flag allows you to enabled/disable that feature of remapping.  Maybe changing the name to include the word 'atlas' would help too.